pkg/storaged/details.jsx

```diff
@@ -73,6 +78,11 @@ export class StdDetailsLayout extends React.Component {
                         </div>
                         <JobsPanel client={this.props.client} />
                     </StackItem>
+                    { drive_ata &&
+                        <StackItem>
+                            <SmartDetails smartInfo={drive_ata} />
+                        </StackItem>
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19103-20230717-115949-00eb709f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)